c-sharp.md

```diff
@@ -5,70 +5,37 @@ tags = ["c#", "csharp", "dotnet", ".net", "webassembly"]
 template = "page_lang"
 [extra]
 author = "Fermyon Staff"
-last_modified = "2022-03-10T21:50:50Z"
+last_modified = "2023-10-26T00:50:50Z"
 ---
 # C# and .NET in WebAssembly
 
 The .NET (aka dotnet) ecosystem supports a variety of languages, including C# (C sharp) and ASP.NET. Programs that target .NET are compiled to an intermediary bytecode format that runs in the .NET Common Language Runtime (CLR). This makes .NET an example of a [virtual machine language](/blog/scripts-vs.compiled-wasm.md).
 
 ## Available Implementations
 
-The .NET ecosystem has long had support for browser-side WebAssembly via the [Blazor toolkit](https://dotnet.microsoft.com/en-us/apps/aspnet/web-apps/blazor).
+The .NET ecosystem has long had support for browser-side WebAssembly via the [Blazor toolkit](https://dotnet.microsoft.com/en-us/apps/aspnet/web-apps/blazor). As of .NET 7, it has support for WebAssembly as a compile target.
```

Comment:
Nit: I think .NET 7 may only have been Steve Sanderson's preview. Not 100% sure though. It's definitely in .NET 8.

Reply:
That's the way I understand it as well. The wasi target is enabled by installing the `wasi-experimental` workload.